modules/ROOT/partials/configuration/autocorrect_capitalize.adoc

```diff
@@ -0,0 +1,22 @@
+[[autocorrect_capitalize]]
+== `+autocorrect_capitalize+`
+
+This option enables or disables capitlization autocorrection when the {productname} editor is loaded. When set to `false`, capitlization autocorrection is disabled when the editor is initialized.
```

Comment:
```suggestion
This option enables or disables capitalization autocorrection when the {productname} editor is loaded. When set to `true`, capitalization autocorrection is disabled when the editor is initialized.
```

Reply:
@tiny-ben-tran , did you mean to change the `false` to `true` here?

Reply:
It is as an example since the default is `false`.

Reply:
Fixed as part of a different commit (that included the whole megilla of new known issues).